txpool/queue.go

```diff
@@ -62,6 +62,20 @@ func (q *accountQueue) prune(nonce uint64) (pruned []*types.Transaction) {
 	return
 }
 
+// clear removes all transactions from the queue.
+func (q *accountQueue) clear() (removed []*types.Transaction) {
+	for {
+		tx := q.pop()
+		if tx == nil {
+			break
+		}
+
+		removed = append(removed, tx)
+	}
+
```

Comment:
Is it possible to clear queue field in accountQueue directly?

Reply:
Actually, it can. What about this approach ?

https://github.com/0xPolygon/polygon-edge/pull/380/commits/c388fac96cebfd3ede0124227c88228f24896c6b